source/_docs/z-wave/controllers.markdown

```diff
@@ -73,32 +73,32 @@ Firmware can be upgraded using the below directions:
   - Z-Wave.Me Razberry 2 (500 series)
   - Z-Wave.Me Razberry 1 (300 series)
 
-If you are just starting out, we recommend that you purchase a 700 series controller or a Raspberry Pi module. The 700 series controllers are the more recent version (when compared to the 500 series). The 700 series controllers support SmartStart, which allows you to add a device by scanning a QR code.
+If you are just starting out, we recommend that you purchase a 700 series adapter or a Raspberry Pi module. The 700 series adapters are the more recent version (when compared to the 500 series). The 700 series adapters support SmartStart, which allows you to add a device by scanning a QR code.
 
 {% tip %}
 It's recommended to use a USB stick, not a module. Passing a module through Docker is more complicated than passing a USB stick through.
 {% endtip %}
 
-## Stick alternatives
+## Stick or module alternatives
 
-The alternative to a stick is a hub that supports Z-Wave. Home Assistant supports the following hubs with Z-Wave support:
+The alternative to a stick or module is a hub that supports Z-Wave. Home Assistant supports the following hubs with Z-Wave support:
```

Comment:
Should we hint that using an adapter with HA directly (and not a 3rd party hub) is recommended for the best experience?